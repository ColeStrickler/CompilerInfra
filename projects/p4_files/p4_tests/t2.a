fn : (arg1: int) -> int {
    a = 4;
    arg1 = a;

    return arg1;
}