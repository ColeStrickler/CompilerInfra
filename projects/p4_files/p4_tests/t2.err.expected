FATAL [2,5]-[2,6]: Undeclared identifier
FATAL [3,12]-[3,13]: Undeclared identifier
Name Analysis Failed